lib/src/network/explorer/explorer_api.dart

```diff
@@ -1535,3 +1535,107 @@ class HashInfo {
         "block_hash": blockHash,
       };
 }
+
+
+class PrioritiesEstimate {
+  PrioritiesEstimate({
+    required this.drtStinky,
```

Comment:
Funnily, this was supposed to be "stingy" but I made a typo and it's now everywhere :rofl: